.gitattributes

```diff
@@ -1,2 +1,2 @@
-vmlinux.h linguist-vendored
+vmlinux*.h linguist-vendored
```

Comment:
We probably need to update this as well.

Reply:
right. Should work :) but it is unneeded now. Thanks